### DISCUSSION TITLE
Removing outdated subscriptions.

### DISCUSSION THREAD

**Discussion by AgentCasey** [writefreely/writefreely#1130] [Questions / Support]
This may not qualify as a bug, however, it is bugging me.  Is there a file in the root, Writefreely, directory, where one can delete subscribers, particularly, subscribers who no longer exists? I've been looking and I cannot come up with an affirmative answer. Or, perhaps, there is an option that can be entered in the config.ini file? Thank you.

**Comment by thebaer** [comment]
Which subscribers are you looking to remove? Email subscribers or fediverse followers?
